go.mod

```diff
@@ -25,8 +25,9 @@ require (
 	k8s.io/api v0.25.0
 	k8s.io/apiextensions-apiserver v0.25.0
 	k8s.io/apimachinery v0.25.0
+	k8s.io/cli-runtime v0.25.0
 	k8s.io/client-go v0.25.0
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
+	k8s.io/kubectl v0.23.4
```

Comment:
can we use [v0.25.0](https://pkg.go.dev/k8s.io/kubectl@v0.25.0) here

Reply:
Updated